node/node.go

```diff
@@ -39,7 +38,7 @@ type Options struct {
 // Node represents the behavior of SSV node
 type Node interface {
 	// Start starts the SSV node
-	Start() error
+	Start(ctx context.Context) error
```

Comment:
SsvNode is holding the ctx, need to remove the ctx here

Reply:
mm I think that's from merging the stage. Should I remote it?

Reply:
yes